2024-05-31 20:39:46,056 [main] WARN  o.a.r.m.MetricRegistriesLoader:81 - Found multiple MetricRegistries implementations: class org.apache.iotdb.consensus.ratis.metrics.MetricRegistryManager, class org.apache.ratis.metrics.impl.MetricRegistriesImpl. Using first found implementation: org.apache.iotdb.consensus.ratis.metrics.MetricRegistryManager@26c47874 
2024-05-31 20:41:15,612 [pool-31-IoTDB-ClientRPC-Processor-1$20240531_124115_00004_1] WARN  o.a.i.d.q.p.e.c.e.ClusterConfigTaskExecutor:1688 - Failed to execute delete timeseries [root.car_01.ms_01, root.car_01.ms_02, root.car_01.ms_03, root.car_01.ms_04] in config node, status is TSStatus(code:508, message:Timeseries [root.car_01.ms_04 ... root.car_01.ms_01] does not exist or is represented by device template). 
2024-05-31 20:41:15,614 [pool-24-IoTDB-MPP-Coordinator-Executor-17] WARN  o.a.i.d.q.p.e.c.ConfigExecution:126 - Failures happened during running ConfigExecution. 
org.apache.iotdb.commons.exception.IoTDBException: Timeseries [root.car_01.ms_04 ... root.car_01.ms_01] does not exist or is represented by device template
	at org.apache.iotdb.db.queryengine.plan.execution.config.executor.ClusterConfigTaskExecutor.deleteTimeSeries(ClusterConfigTaskExecutor.java:1692)
	at org.apache.iotdb.db.queryengine.plan.execution.config.metadata.DeleteTimeSeriesTask.execute(DeleteTimeSeriesTask.java:43)
	at org.apache.iotdb.db.queryengine.plan.execution.config.ConfigExecution.start(ConfigExecution.java:101)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:160)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:174)
	at org.apache.iotdb.db.protocol.thrift.impl.ClientRPCServiceImpl.deleteTimeseries(ClientRPCServiceImpl.java:1399)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteTimeseries.getResult(IClientRPCService.java:4469)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteTimeseries.getResult(IClientRPCService.java:4449)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:38)
	at org.apache.iotdb.db.protocol.thrift.ProcessorWithMetrics.process(ProcessorWithMetrics.java:64)
	at org.apache.thrift.server.TThreadPoolServer$WorkerProcess.run(TThreadPoolServer.java:248)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2024-05-31 20:45:35,032 [pool-31-IoTDB-ClientRPC-Processor-3$20240531_124534_00024_1] WARN  o.a.i.d.q.p.e.c.e.ClusterConfigTaskExecutor:1688 - Failed to execute delete timeseries [root.car_01ms_01, root.car_01ms_02, root.car_01ms_03, root.car_01ms_04] in config node, status is TSStatus(code:508, message:Timeseries [root.car_01ms_01 ... root.car_01ms_04] does not exist or is represented by device template). 
2024-05-31 20:45:35,032 [pool-24-IoTDB-MPP-Coordinator-Executor-12] WARN  o.a.i.d.q.p.e.c.ConfigExecution:126 - Failures happened during running ConfigExecution. 
org.apache.iotdb.commons.exception.IoTDBException: Timeseries [root.car_01ms_01 ... root.car_01ms_04] does not exist or is represented by device template
	at org.apache.iotdb.db.queryengine.plan.execution.config.executor.ClusterConfigTaskExecutor.deleteTimeSeries(ClusterConfigTaskExecutor.java:1692)
	at org.apache.iotdb.db.queryengine.plan.execution.config.metadata.DeleteTimeSeriesTask.execute(DeleteTimeSeriesTask.java:43)
	at org.apache.iotdb.db.queryengine.plan.execution.config.ConfigExecution.start(ConfigExecution.java:101)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:160)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:174)
	at org.apache.iotdb.db.protocol.thrift.impl.ClientRPCServiceImpl.deleteTimeseries(ClientRPCServiceImpl.java:1399)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteTimeseries.getResult(IClientRPCService.java:4469)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteTimeseries.getResult(IClientRPCService.java:4449)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:38)
	at org.apache.iotdb.db.protocol.thrift.ProcessorWithMetrics.process(ProcessorWithMetrics.java:64)
	at org.apache.thrift.server.TThreadPoolServer$WorkerProcess.run(TThreadPoolServer.java:248)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2024-05-31 20:46:56,366 [pool-31-IoTDB-ClientRPC-Processor-5$20240531_124656_00025_1] WARN  o.a.i.d.q.p.e.c.e.ClusterConfigTaskExecutor:370 - Failed to execute delete database [root.car_1] in config node, status is TSStatus(code:508, message:Path [root.car_1] does not exist). 
2024-05-31 20:46:56,366 [pool-24-IoTDB-MPP-Coordinator-Executor-13] WARN  o.a.i.d.q.p.e.c.ConfigExecution:126 - Failures happened during running ConfigExecution. 
org.apache.iotdb.commons.exception.IoTDBException: Path [root.car_1] does not exist
	at org.apache.iotdb.db.queryengine.plan.execution.config.executor.ClusterConfigTaskExecutor.deleteDatabase(ClusterConfigTaskExecutor.java:374)
	at org.apache.iotdb.db.queryengine.plan.execution.config.metadata.DeleteStorageGroupTask.execute(DeleteStorageGroupTask.java:41)
	at org.apache.iotdb.db.queryengine.plan.execution.config.ConfigExecution.start(ConfigExecution.java:101)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:160)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:174)
	at org.apache.iotdb.db.protocol.thrift.impl.ClientRPCServiceImpl.deleteStorageGroups(ClientRPCServiceImpl.java:1442)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteStorageGroups.getResult(IClientRPCService.java:4494)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteStorageGroups.getResult(IClientRPCService.java:4474)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:38)
	at org.apache.iotdb.db.protocol.thrift.ProcessorWithMetrics.process(ProcessorWithMetrics.java:64)
	at org.apache.thrift.server.TThreadPoolServer$WorkerProcess.run(TThreadPoolServer.java:248)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2024-05-31 20:47:28,720 [pool-31-IoTDB-ClientRPC-Processor-6$20240531_124728_00026_1.1.0] WARN  o.a.i.d.q.p.s.FragmentInstanceDispatcherImpl:392 - write locally failed. TSStatus: TSStatus(code:302, subStatus:[TSStatus(code:503, message:Path [root.car_01.ms_01] already exist), TSStatus(code:503, message:Path [root.car_01.ms_02] already exist), TSStatus(code:503, message:Path [root.car_01.ms_03] already exist), TSStatus(code:503, message:Path [root.car_01.ms_04] already exist)]), message: null 
2024-05-31 20:48:04,050 [pool-31-IoTDB-ClientRPC-Processor-7$20240531_124803_00045_1] WARN  o.a.i.d.q.p.e.c.e.ClusterConfigTaskExecutor:1688 - Failed to execute delete timeseries [root.car_01ms_01, root.car_01ms_02, root.car_01ms_03, root.car_01ms_04] in config node, status is TSStatus(code:508, message:Timeseries [root.car_01ms_01 ... root.car_01ms_04] does not exist or is represented by device template). 
2024-05-31 20:48:04,050 [pool-24-IoTDB-MPP-Coordinator-Executor-4] WARN  o.a.i.d.q.p.e.c.ConfigExecution:126 - Failures happened during running ConfigExecution. 
org.apache.iotdb.commons.exception.IoTDBException: Timeseries [root.car_01ms_01 ... root.car_01ms_04] does not exist or is represented by device template
	at org.apache.iotdb.db.queryengine.plan.execution.config.executor.ClusterConfigTaskExecutor.deleteTimeSeries(ClusterConfigTaskExecutor.java:1692)
	at org.apache.iotdb.db.queryengine.plan.execution.config.metadata.DeleteTimeSeriesTask.execute(DeleteTimeSeriesTask.java:43)
	at org.apache.iotdb.db.queryengine.plan.execution.config.ConfigExecution.start(ConfigExecution.java:101)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:160)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:174)
	at org.apache.iotdb.db.protocol.thrift.impl.ClientRPCServiceImpl.deleteTimeseries(ClientRPCServiceImpl.java:1399)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteTimeseries.getResult(IClientRPCService.java:4469)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteTimeseries.getResult(IClientRPCService.java:4449)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:38)
	at org.apache.iotdb.db.protocol.thrift.ProcessorWithMetrics.process(ProcessorWithMetrics.java:64)
	at org.apache.thrift.server.TThreadPoolServer$WorkerProcess.run(TThreadPoolServer.java:248)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2024-05-31 20:48:36,324 [pool-31-IoTDB-ClientRPC-Processor-8$20240531_124836_00046_1.1.0] WARN  o.a.i.d.q.p.s.FragmentInstanceDispatcherImpl:392 - write locally failed. TSStatus: TSStatus(code:302, subStatus:[TSStatus(code:503, message:Path [root.car_01.ms_01] already exist), TSStatus(code:503, message:Path [root.car_01.ms_02] already exist), TSStatus(code:503, message:Path [root.car_01.ms_03] already exist), TSStatus(code:503, message:Path [root.car_01.ms_04] already exist)]), message: null 
2024-05-31 20:49:11,688 [pool-31-IoTDB-ClientRPC-Processor-9$20240531_124911_00065_1] WARN  o.a.i.d.q.p.e.c.e.ClusterConfigTaskExecutor:1688 - Failed to execute delete timeseries [root.car_01ms_01, root.car_01ms_02, root.car_01ms_03, root.car_01ms_04] in config node, status is TSStatus(code:508, message:Timeseries [root.car_01ms_01 ... root.car_01ms_04] does not exist or is represented by device template). 
2024-05-31 20:49:11,688 [pool-24-IoTDB-MPP-Coordinator-Executor-16] WARN  o.a.i.d.q.p.e.c.ConfigExecution:126 - Failures happened during running ConfigExecution. 
org.apache.iotdb.commons.exception.IoTDBException: Timeseries [root.car_01ms_01 ... root.car_01ms_04] does not exist or is represented by device template
	at org.apache.iotdb.db.queryengine.plan.execution.config.executor.ClusterConfigTaskExecutor.deleteTimeSeries(ClusterConfigTaskExecutor.java:1692)
	at org.apache.iotdb.db.queryengine.plan.execution.config.metadata.DeleteTimeSeriesTask.execute(DeleteTimeSeriesTask.java:43)
	at org.apache.iotdb.db.queryengine.plan.execution.config.ConfigExecution.start(ConfigExecution.java:101)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:160)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:174)
	at org.apache.iotdb.db.protocol.thrift.impl.ClientRPCServiceImpl.deleteTimeseries(ClientRPCServiceImpl.java:1399)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteTimeseries.getResult(IClientRPCService.java:4469)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteTimeseries.getResult(IClientRPCService.java:4449)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:38)
	at org.apache.iotdb.db.protocol.thrift.ProcessorWithMetrics.process(ProcessorWithMetrics.java:64)
	at org.apache.thrift.server.TThreadPoolServer$WorkerProcess.run(TThreadPoolServer.java:248)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2024-05-31 20:49:23,996 [pool-31-IoTDB-ClientRPC-Processor-10$20240531_124923_00084_1] WARN  o.a.i.d.q.p.e.c.e.ClusterConfigTaskExecutor:1688 - Failed to execute delete timeseries [root.car_01ms_01, root.car_01ms_02, root.car_01ms_03, root.car_01ms_04] in config node, status is TSStatus(code:508, message:Timeseries [root.car_01ms_01 ... root.car_01ms_04] does not exist or is represented by device template). 
2024-05-31 20:49:23,996 [pool-24-IoTDB-MPP-Coordinator-Executor-5] WARN  o.a.i.d.q.p.e.c.ConfigExecution:126 - Failures happened during running ConfigExecution. 
org.apache.iotdb.commons.exception.IoTDBException: Timeseries [root.car_01ms_01 ... root.car_01ms_04] does not exist or is represented by device template
	at org.apache.iotdb.db.queryengine.plan.execution.config.executor.ClusterConfigTaskExecutor.deleteTimeSeries(ClusterConfigTaskExecutor.java:1692)
	at org.apache.iotdb.db.queryengine.plan.execution.config.metadata.DeleteTimeSeriesTask.execute(DeleteTimeSeriesTask.java:43)
	at org.apache.iotdb.db.queryengine.plan.execution.config.ConfigExecution.start(ConfigExecution.java:101)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:160)
	at org.apache.iotdb.db.queryengine.plan.Coordinator.execute(Coordinator.java:174)
	at org.apache.iotdb.db.protocol.thrift.impl.ClientRPCServiceImpl.deleteTimeseries(ClientRPCServiceImpl.java:1399)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteTimeseries.getResult(IClientRPCService.java:4469)
	at org.apache.iotdb.service.rpc.thrift.IClientRPCService$Processor$deleteTimeseries.getResult(IClientRPCService.java:4449)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:38)
	at org.apache.iotdb.db.protocol.thrift.ProcessorWithMetrics.process(ProcessorWithMetrics.java:64)
	at org.apache.thrift.server.TThreadPoolServer$WorkerProcess.run(TThreadPoolServer.java:248)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2024-05-31 21:14:10,025 [IoTDB-Shutdown-Hook] WARN  o.a.i.c.conf.CommonConfig:425 - Change system status to ReadOnly! Only query statements are permitted! 
2024-05-31 21:14:12,910 [IoTDB-Shutdown-Hook] WARN  o.a.i.d.p.c.ConfigNodeClient:339 - Failed to connect to ConfigNode TEndPoint(ip:127.0.0.1, port:10710) from DataNode TEndPoint(ip:0.0.0.0, port:6667) when executing reportDataNodeShutdown, Exception: 
org.apache.thrift.transport.TTransportException: java.net.SocketException: Connection reset by peer
	at org.apache.thrift.transport.TIOStreamTransport.flush(TIOStreamTransport.java:213)
	at org.apache.iotdb.rpc.TElasticFramedTransport.flush(TElasticFramedTransport.java:141)
	at org.apache.thrift.TServiceClient.sendBase(TServiceClient.java:73)
	at org.apache.thrift.TServiceClient.sendBase(TServiceClient.java:62)
	at org.apache.iotdb.confignode.rpc.thrift.IConfigNodeRPCService$Client.send_reportDataNodeShutdown(IConfigNodeRPCService.java:1092)
	at org.apache.iotdb.confignode.rpc.thrift.IConfigNodeRPCService$Client.reportDataNodeShutdown(IConfigNodeRPCService.java:1084)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient.lambda$reportDataNodeShutdown$6(ConfigNodeClient.java:404)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient.executeRemoteCallWithRetry(ConfigNodeClient.java:328)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient.reportDataNodeShutdown(ConfigNodeClient.java:403)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient$$EnhancerByCGLIB$$c22e8340.CGLIB$reportDataNodeShutdown$4(<generated>)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient$$EnhancerByCGLIB$$c22e8340$$FastClassByCGLIB$$6e036d4b.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228)
	at org.apache.iotdb.commons.client.sync.SyncThriftClientWithErrorHandler.intercept(SyncThriftClientWithErrorHandler.java:53)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient$$EnhancerByCGLIB$$c22e8340.reportDataNodeShutdown(<generated>)
	at org.apache.iotdb.db.service.IoTDBShutdownHook.run(IoTDBShutdownHook.java:125)
Caused by: java.net.SocketException: Connection reset by peer
	at java.base/sun.nio.ch.SocketDispatcher.write0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.write(SocketDispatcher.java:54)
	at java.base/sun.nio.ch.NioSocketImpl.tryWrite(NioSocketImpl.java:394)
	at java.base/sun.nio.ch.NioSocketImpl.implWrite(NioSocketImpl.java:410)
	at java.base/sun.nio.ch.NioSocketImpl.write(NioSocketImpl.java:440)
	at java.base/sun.nio.ch.NioSocketImpl$2.write(NioSocketImpl.java:819)
	at java.base/java.net.Socket$SocketOutputStream.write(Socket.java:1195)
	at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:125)
	at java.base/java.io.BufferedOutputStream.implFlush(BufferedOutputStream.java:252)
	at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:240)
	at org.apache.thrift.transport.TIOStreamTransport.flush(TIOStreamTransport.java:208)
	... 14 common frames omitted
2024-05-31 21:14:14,914 [IoTDB-Shutdown-Hook] WARN  o.a.t.t.TIOStreamTransport:158 - Error closing output stream. 
java.net.SocketException: Socket closed
	at java.base/sun.nio.ch.NioSocketImpl.ensureOpenAndConnected(NioSocketImpl.java:163)
	at java.base/sun.nio.ch.NioSocketImpl.beginWrite(NioSocketImpl.java:362)
	at java.base/sun.nio.ch.NioSocketImpl.implWrite(NioSocketImpl.java:407)
	at java.base/sun.nio.ch.NioSocketImpl.write(NioSocketImpl.java:440)
	at java.base/sun.nio.ch.NioSocketImpl$2.write(NioSocketImpl.java:819)
	at java.base/java.net.Socket$SocketOutputStream.write(Socket.java:1195)
	at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:125)
	at java.base/java.io.BufferedOutputStream.implFlush(BufferedOutputStream.java:252)
	at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:240)
	at java.base/java.io.FilterOutputStream.close(FilterOutputStream.java:184)
	at org.apache.thrift.transport.TIOStreamTransport.close(TIOStreamTransport.java:156)
	at org.apache.thrift.transport.TSocket.close(TSocket.java:252)
	at org.apache.iotdb.rpc.TElasticFramedTransport.close(TElasticFramedTransport.java:97)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient.tryToConnect(ConfigNodeClient.java:250)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient.waitAndReconnect(ConfigNodeClient.java:230)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient.executeRemoteCallWithRetry(ConfigNodeClient.java:342)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient.reportDataNodeShutdown(ConfigNodeClient.java:403)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient$$EnhancerByCGLIB$$c22e8340.CGLIB$reportDataNodeShutdown$4(<generated>)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient$$EnhancerByCGLIB$$c22e8340$$FastClassByCGLIB$$6e036d4b.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228)
	at org.apache.iotdb.commons.client.sync.SyncThriftClientWithErrorHandler.intercept(SyncThriftClientWithErrorHandler.java:53)
	at org.apache.iotdb.db.protocol.client.ConfigNodeClient$$EnhancerByCGLIB$$c22e8340.reportDataNodeShutdown(<generated>)
	at org.apache.iotdb.db.service.IoTDBShutdownHook.run(IoTDBShutdownHook.java:125)
2024-05-31 21:14:14,917 [IoTDB-Shutdown-Hook] WARN  o.a.i.d.p.c.ConfigNodeClient:261 - The current node may have been down TEndPoint(ip:127.0.0.1, port:10710),try next node 
2024-05-31 21:14:14,929 [IoTDB-Shutdown-Hook] WARN  o.a.i.d.p.c.ConfigNodeClient:261 - The current node may have been down TEndPoint(ip:127.0.0.1, port:10710),try next node 
